generators/lib/distros/octane.js

```diff
@@ -31,21 +32,29 @@ function init() {
     done();
   };
 
-  module.modifyComposer = function(yo, options, localComposer, isNewProject, done) {
+  module.loadComposer = function(yo, options) {
+    var composer = {};
+    var file = yo.templatePath('drupal/drupal/' + options.drupalDistroRelease + '/composer.json');
```

Comment:
The only reason why we have the loadComposer helper is because Octane is using Drupal's composer.json template. The only reason it's different otherwise is that Octane should never really fall back to using the Drupal 7 composer.json the gdt template directory provides.